### PR TITLE
add template for compound forks

### PR DESCRIPTION
**Pull request by 0xbe1**
refer to README for details

### CODE REVIEW

File: subgraphs/compound-forks/src/constants.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+
+////////////////////////
+///// Schema Enums /////
+////////////////////////
+
+// The enum values are derived from Coingecko slugs (converted to uppercase
+// and replaced hyphens with underscores for Postgres enum compatibility)
+export namespace Network {
```
Comment (this-username-is-taken) [inline subgraphs/compound-forks/src/constants.ts:9]: Update

File: subgraphs/compound-forks/src/mapping.ts [hunk 2]
```diff
@@ -0,0 +1,1445 @@
...
+): BigDecimal {
+  let mantissaDecimalFactor = 18 - underlyingDecimals + 18;
+  let bdFactor = exponentToBigDecimal(mantissaDecimalFactor);
+  return getOrElse<BigInt>(getUnderlyingPriceResult, BIGINT_ZERO)
+    .toBigDecimal()
+    .div(bdFactor);
+}
+
+function getMarketHourlySnapshotID(marketID: string, timestamp: i32): string {
+  return marketID
```
Comment (this-username-is-taken) [inline subgraphs/compound-forks/src/mapping.ts:1415]: This is actually changed in schema 1.2.1

File: subgraphs/compound-forks/src/constants.ts (hunk 1 above)
Comment (0xbe1) [reply subgraphs/compound-forks/src/constants.ts:11]: updated

File: subgraphs/compound-forks/src/mapping.ts (hunk 2 above)
Comment (0xbe1) [reply subgraphs/compound-forks/src/mapping.ts]: updated to 1.2.1

**Comment by this-username-is-taken** [comment]
LGTM!

**Pull request merged**